targets/ramips-mt76x8

```diff
@@ -125,3 +125,7 @@ device('xiaomi-mi-router-4a-100m-international-edition', 'xiaomi_mi-router-4a-10
 device('xiaomi-mi-router-4c', 'xiaomi_mi-router-4c', {
 	factory = false,
 })
+
+device('xiaomi-mi-ra75', 'xiaomi_mi-ra75', {
```

Comment:
As per out policy, the device needs to be flashable without opening the case.

Please mark it as broken with a comment on the flashing-method. Maybe we can then enable these in the future.

Reply:
Done